python-rhsm.spec

```diff
@@ -38,6 +38,7 @@ Requires: python-dateutil
 %if %use_simplejson
 Requires: python-simplejson
 %endif
+Requires: python-rhsm-certificates
```

Comment:
Lock this to the current version so that all the python-rhsm packages will upgrade together:

```
Requires: python-rhsm-certificates = %{version}-%{release}
```